src/test/java/htsjdk/samtools/ValidateSamFileTest.java

```diff
@@ -528,7 +528,7 @@ public void testHeaderVersions() throws Exception {
         }
 
         // Test an unacceptable version
-        testHeaderVersion("1.6", false);
+        testHeaderVersion("1.7", false);
```

Comment:
If you used `1.1` instead then you'd no longer need to update it when the version was changed.
```suggestion
        testHeaderVersion("1.1", false);
```